content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+In any scenario where a self-signed TLS certificate is used, you need to import the self-signed CA certificate into the OS system.
+
+You can use the following command:
+
+```shell
+keytool -list -v -cacerts
+```
+
+to look at things like CA certificates in your JVM.
+
+You'll find something like the following:
+
+```dart
+Alias name: digicertassuredidrootca
+Creation date: Feb 2, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+Issuer: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+...
+```
+
+This is a recognized CA certificate issued by Digicert.
+
+So, after you use `keytool` to import /opt/mapr/conf/ca/chain-ca.pem into the JVM, use keytool -list -v -cacerts to see something similar to the following:
+
+```dart
+Alias name: edf-clustera-ca
+Creation date: Feb 16, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=MapR Engineering Signing CA, OU=MapR Engineering Signing CA, O=MapR, DC=hpecorp, DC=net
+Issuer: CN=MapR Engineering Root CA, OU=MapR Engineering Root CA, O=MapR, DC=hpecorp, DC=net
+...
+```
+
+#### 2. The documentation describes several client software that you can use to interact with the HPE Ezmeral Object Store. I would like to add the following clarification:
+
+When using the `mc` command-line tool or the `s3cmd` command-line tool to interact with HPE Ezmeral Data Fabric Object Store without completing the configuration of "[Enabling S3 Virtual-Host-Style Requests](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_nnp_kr2_bvb)", some commands will not work properly.
+
+In this case, for management operations, such as creating accounts, IAM accounts, buckets, etc., I recommend you use the `mc` command line tool.
+
+For object listing, getting, putting, and deleting operations, I recommend you use `AWS CLI`.
+
+#### 3. Cannot revert to HTTP mode after enabling HTTPS.
+
+If the Object Store was installed using the Installer, the Object Store would also have security enabled, including HTTPS.
+
+Although you can see the below description:
+
+> To revert to http access, comment out the moss.certs.dir=/opt/mapr/conf line in the /opt/mapr/conf/moss.conf file.
+
+from here - [HTTP Access to Object Store](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_bg1_zd1_vsb), but this configuration change alone does not change Object Store to HTTP mode, and there is nothing else in HPE Ezmeral Data Fabric's documentation on how to modify Object Store's TLS mode.
+
+### [](#create-a-bucket-in-hpe-ezmeral-data-fabric-object-store-and-upload-some-objects)Create a bucket in HPE Ezmeral Data Fabric Object Store and upload some objects
+
+In order to create a Bucket, you need to create an Account, an IAM User, and finally a Bucket in sequence.
```

Comment:
Change "Bucket" to "bucket" in all places and "Account" to "account".

I'm also questioning "Access Policy". Shouldn't this be "...you also need to prepare access policy control for who (IAM User) can perform..." ?